src/index.js

```diff
@@ -4,12 +4,16 @@ import { BrowserRouter } from "react-router-dom";
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import { Provider } from "react-redux";
+import store from "../src/redux/store";
```

Comment:
Why did you traverse outside the `src` folder?
Isn't this supposed to be

```suggestion
import store from "./redux/store";
```